packages/strapi-hook-knex/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "main": "./lib",
   "dependencies": {
-    "knex": "^0.13.0",
+    "knex": "^0.15.2",
```

Comment:
Verify there are no critical breaking changes before merging

Reply:
@lauriejim can you confirm that this is not a breaking change?

Reply:
Some entries from the `knex` change logs:

*v0.14.0*:
Updated allowed operator list with some missing operators and make all to lower case

*v0.15.0*:
json data type is no longer converted to text within a schema builder migration for MySQL databases (note that JSON data type is only supported for MySQL 5.7.8+)

Drop mariadb support

We need to test extensively in a test branch before updating knex this many versions at a time.

Reply:
Knex change log: https://github.com/tgriesser/knex/blob/master/CHANGELOG.md